test/test_utils.ml

```diff
@@ -214,5 +264,6 @@ let suite =
            "util_safe_html" >:: util_safe_html;
            "util_string_with_macros" >:: util_string_with_macros;
            "util_transl_a_of_b" >:: util_transl_a_of_b;
+           "util_eval_translate" >:: util_eval_translate;
          ];
```

Comment:
thanks for adding tests!!

for tests maybe the alcotest PR which is mergeable now should be merged before those new test, you will have to change some stuff but it should not be too hard

Reply:
also if you manage to write test for Cousins it would be nice

Reply:
show me what needs to be changed